src/main/kotlin/blackJack/domain/People.kt

```diff
@@ -0,0 +1,19 @@
+package blackJack.domain
+
+open class People(val name: String) {
```

Comment:
`People`은 인스턴스화하지(instantiate) 않으니 추상 클래스를 사용하면 어떨까요?

```suggestion
abstract class People(val name: String) {
```